### PR TITLE
Add import of simpleapi to scripts generated by script builder

### PR DESCRIPTION
**Pull request by Harrietbrown**
This PR adds the import of `mantid.simpleapi` to scripts generated by the algorithm history interface

**To test:**

1. Load a worksapce and perform varius algorithms (rebin, crop, ect.)
2. right click on workspace and open History
3. click on copy to clipboard
4. open a new script in the editor and clear the default inports.
5. paste the copied script and check that the fist line is `from mantid.simplaeapi import *`

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/techniques/index.rst [hunk 1]
```diff
@@ -17,3 +17,4 @@ Techniques
    :maxdepth: 1
 
    *
+test_workspacehistorygeneration
```
Comment (DanielMurphy22) [inline docs/source/techniques/index.rst:20]: Should this be here?

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
Very good, the import line is added and thanks for updating the tests! I'm not sure the change to the techniques docs page should be included.

I found a small niggle using the script save to file. If I open the History dialog on a workspace and click Script to File, then type in a filename (importantly I did not put .py in my filename, but the filetype is set to .py) and then click Save. The file is annoying saved without the .py extension so it cannot be searched for when trying to open a script in MantidWorkbench. If I use File > Save Script As then it will successfully add the .py extension to the file for me.
Screenshot 2022-09-28 at 13 42 39

You may decide this next bit is a little beyond scope. If you'd like me to write up a separate issue then tell me!

File: docs/source/techniques/index.rst (hunk 1 above)
Comment (Harrietbrown) [reply docs/source/techniques/index.rst:20]: Thank's for spotting this. Funny how these things can slip in.

**Comment by Harrietbrown** [comment]
The changes in this pr are to the underlying history generating algorithms, not the worksapce history interface per say. I would write up the missing file extention as a separate issue.

**Review by DanielMurphy22 — APPROVED** [review]
Good job!

**Pull request merged**